main.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+
+from app.modbus_client import ModbusClient
+from app.mqtt_client import MqttClient
+from app.configuration import Configuration
+
+SLAVE=0x01
+
+BROKER_HOST = os.getenv('BROKER_HOST')
```

Comment:
this should be part of the configuration file, in the YAML file as well.
Essentially everything thats dynamic, should be part of the YAML config